examples/collectionless-item.json

```diff
@@ -74,24 +75,34 @@
   "links": [
     {
       "rel": "self",
-      "href": "http://cool-sat.com/catalog/CS3-20160503_132130_04/CS3-20160503_132130_04.json"
+      "href": "https://raw.githubusercontent.com/radiantearth/stac-spec/examples-refresh/examples/simple-item.json",
+      "type": "application/geo+json"
     },
     {
-      "rel": "root",
-      "href": "http://cool-sat.com/catalog/catalog.json"
+      "rel": "collection",
+      "href": "./collection.json",
+      "type": "application/json",
+      "title": "Simple Example Collection"
     },
     {
-      "rel": "parent",
-      "href": "http://cool-sat.com/catalog/CS3-20160503_132130_04/catalog.json"
+      "rel": "root",
+      "href": "./collection.json",
+      "type": "application/json"
     },
     {
-      "rel": "collection",
-      "href": "http://cool-sat.com/catalog/CS3-20160503_132130_04/catalog.json"
+      "rel": "root",
+      "href": "./collection.json",
+      "type": "application/json"
     },
     {
       "rel": "alternate",
       "type": "text/html",
       "href": "http://cool-sat.com/catalog/CS3-20160503_132130_04/CS3-20160503_132130_04.html"
```

Comment:
Should we add the root catalog to STAC Index and then we have real HTML Urls...

Reply:
Maybe? But I think we should get it set up with the 'publishing' mechanism first, so that it's a final, stable location, and not one we have to update.

Reply:
Is the version number update really a problem? Whether search & replace works on 100 or 200 matches in the files don't really care, right? If it makes it work on GH only, maybe it's enough? Otherwise, you need to implement CI and scripts again for deployment.

Reply:
I don't think the version update is really a problem. I was leaning more towards just do it in github, but Matt liked the idea of publishing them to an online location, so I made #984 for him. But I see that as 'nice to have', and the default to me is just to keep it as is in github, and update the version. If we decide **not** to do the publishing thing then I'm definitely cool to just call it good in github and set the html rel=alternates to stacindex based on this. I just want to be sure that's the route we're taking.